### PR TITLE
Write user stories & draw wireframes

### PR DESCRIPTION
**Pull request by raphiz**
@fabianhauser As discussed, would be nice if you can skim this one and give me feedback if the current form is appropriate.

### CODE REVIEW

**Comment by fabianhauser** [comment]
@raphiz I like the current form. Maybe we could group the requirements into groups, so that it is a bit easier to get an overview and find things :)

**Comment by raphiz** [comment]
@fabianhauser Thanks for the feedback. Any suggestions for the kind of groups?

**Comment by fabianhauser** [comment]
@raphiz what about:
- Authentication (1,2)
- Topic Management (3-5)
- Topic Subscriptions (6-8)
- Persistent Items (9,10)
- Validation? (11)

**Comment by raphiz** [comment]
Thanks for the groups. I totally agree with the first and the last. The three in-between are all kind of "topic management" and i therefore kept it "verbose" for now.

File: documents/final-submission-document/appendices/requirements.tex [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+\subsection{Create a New Topic}
+
+As an Administrator,\\
+I want to create a new Topic on the associated Controller\\
+so that I and not tied to a fixed set of Topics.
+
+\subsubsection{Override Default Topic Configuration}
+
+As an Administrator in the process of creating a new Topic,\\
+I want to override the default configuration (e.g. the subscription model) \\
```
[the remark this answers is not in the document]
Comment (raphiz) [reply documents/final-submission-document/appendices/requirements.tex]: As discussed, I will change subscription model  to "affiliations".

**Pull request merged**